README.md

```diff
@@ -31,7 +31,7 @@ The steps are:
 From [JavaScript example](./examples/javascript.js).
 
 ```js
-const {toMessagePublishedData} = require('@google/events/src/google/events/cloud/pubsub/v1');
+const {toMessagePublishedData} = require('@google/events/cloud/pubsub/v1');
```

Comment:
🎉 Those few characters are small but removing them makes it so much better, and shows that we're paying attention to the details.